src/palace/manager/celery/monitoring.py

```diff
@@ -182,20 +181,25 @@ def on_shutter(self, state: State) -> None:
         for task in state.tasks.values():
             try:
                 tasks[task.name].update(task)
-                self.queues[task.routing_key].update(task)
-            except KeyError:
-                self.logger.exception(
-                    "Error processing task %s with routing key %s",
-                    task.name,
-                    task.routing_key,
+                for queue in self.queues:
+                    queue.update(task)
+            except:
+                task_info = ", ".join(
+                    [
+                        f"[{k}]:{v}"
+                        for k, v in task.info(
+                            extra=["name", "sent", "started", "uuid"]
+                        ).items()
+                    ]
                 )
+                self.logger.exception(f"Error processing task: {task_info}.")
```

Comment:
It might be useful to track what exceptions we're getting here, in case we eventually want to narrow the reach of the `except` above.

Reply:
`logger.exception` logs the full exception stack trace, along with the information about the exception.